node/node.go

```diff
@@ -971,6 +971,15 @@ func (n *Node) superviseManager(ctx context.Context, securityConfig *ca.Security
 	}
 }
 
+// Downgrade reverts the node key to older format so that it can be
+// run on older version of swarmkit
+func (n *Node) Downgrade() error {
```

Comment:
Apologies for the nitpick, similar to the comment about the command line tool being "downgrade-key" instead of just "downgrade", since this is a function on a node itself, would it make sense for this function to also be called "DowngradeKey"?

Reply:
Resolved! :)